chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -0,0 +1,41 @@
+#include <vector>
+
+#include "chainerx/array.h"
+#include "chainerx/backend.h"
+#include "chainerx/backprop_mode.h"
+#include "chainerx/backward_builder.h"
+#include "chainerx/backward_context.h"
+#include "chainerx/constant.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/graph.h"
+#include "chainerx/kernels/math.h"
+#include "chainerx/macro.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/math.cc"
+#include "chainerx/routines/math.h"
+#include "chainerx/routines/manipulation.h"
+#include "chainerx/routines/type_util.h"
+#include "chainerx/scalar.h"
+#include "chainerx/shape.h"
+#include "chainerx/strides.h"
+
+namespace chainerx {
+Array ClippedRelu(const Array& x, Scalar z) {
+    return Minimum(Maximum(0, x), z);
+}
+
+Array Crelu(const Array& x, nonstd::optional<int8_t> axis) {
```

Comment:
I think Crelu can be implement with a single memory allocation similar to `F.crelu` in Chainer. Could you write a TODO to optimize the implementation? You are of course welcome to implement it that way if you'd like to.